### ISSUE TITLE
.exit does not exit from REPL

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/taiko#519]
### Expected behavior
.exit should close the browser and exits from the REPL

### Actual behavior
It does not exit from REPL

### Steps to reproduce
1. openBrowser()
2. goto("google.com")

### Versions
```
Version: 0.7.0 (Chromium:74.0.3723.0)
commit: 6d41672657285488ba74b4414dfe770493279b2c
```

**Comment by shubhamsc** [comment]
Fix available in a5565a587804d74ea5b7f10f3df844118541522b

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed. Tested with the version ```a5565a5 ```
